fpdcc/document-search
# Allow for license number to be any string

## Overview

This removes the validator from the `license_number` field to allow it to be a string of any characters.

- Closes #110 
- Closes #113

## Testing Instructions

* Run migrations and bring up the container
* Create/edit a license
* Enter in a license number without the previous requirement (a string that started with "O-")
* Confirm that it does not return any validation errors for that
